api/src/main/java/liquibase/ext/sqlgenerator/core/ModifyColumnGenerator.java

```diff
@@ -192,13 +192,11 @@ String getPostDataTypeString(Database database) {
 	String getDefaultClause(ColumnConfig column, Database database) {
 		String clause = "";
 		String defaultValue = column.getDefaultValue();
-		if (defaultValue != null) {
-			if (database instanceof MySQLDatabase) {
-				clause += " DEFAULT " + DataTypeFactory.getInstance().fromObject(defaultValue, database)
-					.objectToSql(defaultValue, database);
-			}
-		}
-		return clause;
-	}
+		 if (defaultValue != null && database instanceof MySQLDatabase) {
+	      clause += " DEFAULT "
+	          + DataTypeFactory.getInstance().fromObject(defaultValue, database).objectToSql(defaultValue, database);
+	    }
+	    return clause;
+	  }
```

Comment:
Please write to me, do you have the option to enable whitespace visibility in your IDE? Then it will be much easier, because as you can see you still have a mixture of tabs and spaces.
![image](https://user-images.githubusercontent.com/9057882/91315331-dc57e280-e7b7-11ea-8566-8e81f030ed41.png)

Reply:
Which IDE are you using to track those spaces?

Reply:
I use IDEA IntelliJ for Java, but I also write in Visual Studio Code sometimes. Both have the ability to display whitespace, so I think your IDE can do that too.